R/validation_asserts.R

```diff
@@ -94,6 +100,30 @@ assert_consistent_regions <- function(shape_regions, test_regions, test_source)
   invisible(TRUE)
 }
 
+assert_consistent_region_codes <- function(pjnz_codes, shape_codes) {
+  if (!setequal(pjnz_codes, shape_codes)) {
+    missing_shape_codes <- setdiff(shape_codes, pjnz_codes)
+    missing_pjnz_codes <- setdiff(pjnz_codes, shape_codes)
+    if (length(missing_shape_codes) > 0 &&
+        length(missing_pjnz_codes) > 0) {
+      separator <- "\n"
+    } else {
+      separator <- ""
+    }
+    stop(sprintf(
```

Comment:
This could be simplified - this message in particular, but the whole function in general

Reply:
Have tidied that now